addons/account/models/account_journal_dashboard.py

```diff
@@ -677,19 +677,8 @@ def open_action(self):
         action['context'] = context
         action['context'].update({
             'default_journal_id': self.id,
-            'search_default_journal_id': self.id,
         })
 
-        domain_type_field = action['res_model'] == 'account.move.line' and 'move_id.move_type' or 'move_type' # The model can be either account.move or account.move.line
-
-        # Override the domain only if the action was not explicitly specified in order to keep the
-        # original action domain.
-        if not self._context.get('action_name'):
-            if self.type == 'sale':
-                action['domain'] = [(domain_type_field, 'in', ('out_invoice', 'out_refund', 'out_receipt'))]
-            elif self.type == 'purchase':
-                action['domain'] = [(domain_type_field, 'in', ('in_invoice', 'in_refund', 'in_receipt', 'entry'))]
-
```

Comment:
How is this related ?

Reply:
My bad, I didn't realize why this was necessary but I now get this is needed when opening the action via kanban card > 3 dots > (Invoices|CreditNote...)